gui/objects_map/wallet_names.py

```diff
@@ -19,6 +19,7 @@
 savedAddressView_Delegate = {"container": mainWallet_Saved_Addreses_List, "objectName": RegularExpression("savedAddressView_Delegate*"), "type": "SavedAddressesDelegate", "visible": True}
 send_StatusRoundButton = {"container": "", "type": "StatusRoundButton", "unnamed": 1, "visible": True}
 savedAddressView_Delegate_menuButton = {"container": mainWindow_SavedAddressesView, "objectName": RegularExpression("savedAddressView_Delegate_menuButton*"), "type": "StatusRoundButton", "visible": True}
+savedAddresses_area = {"container": mainWindow_SavedAddressesView, "type": "SavedAddresses", "unnamed": 1, "visible": True}
```

Comment:
in case we can add an object name for that - i think we should go with it

Reply:
Added) Created PR - https://github.com/status-im/status-desktop/pull/13216 and also changed in object map